src/Validation/Validation.php

```diff
@@ -1167,6 +1167,8 @@ public static function range($check, $lower = null, $upper = null)
      * - an optional query string (get parameters)
      * - an optional fragment (anchor tag) as defined in RFC 3986
      *
+     * Does not allow local network addresses, such as "localhost".
```

Comment:
This could be added in the bullet point above regarding valid domain names, instead of a separate statement.

Reply:
This note is less about what the validation method checks but a special hint about what it does not allow.
Would prefer it to be separate, as it increases visiblity.